integration/nevermined/Permissions.test.ts

```diff
@@ -0,0 +1,117 @@
+import { assert } from 'chai'
+import { config } from '../config'
+import { Nevermined, Account, NewPermission, PermissionType } from '../../src'
+import { sleep } from '../utils/utils'
+
+describe('Permissions', () => {
+    let nevermined: Nevermined
+    let account1: Account
+    let account2: Account
+    let newPermission: NewPermission
+    let id: string
+
+    before(async () => {
+        nevermined = await Nevermined.getInstance(config)
+        ;[account1, account2] = await nevermined.accounts.list()
+
+        config.marketplaceAuthToken =
+            'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiIweGUyREQwOWQ3MTlEYTg5ZTVhM0QwRjI1NDljN0UyNDU2NmU5NDcyNjAiLCJzdWIiOiJ1LWU2YzI2NDhjLTIwZjktNDJlMC1iMWZlLWZjZWEwNzA4ODY3NyIsInJvbGVzIjpbImFkbWluIl0sImlhdCI6MTY1MTI0ODM1NCwiZXhwIjoxNzUxMjUxOTU0fQ.p9fr_c_HVlJzY1cJSGDod1zMdhRCRWdExOB_UxMDrKg'
```

Comment:
where does this come from? also don't know if this will always work since the token has an expiration time.
It would be best to generate one on the fly, no?

Reply:
for that we need to create that account which also is needed an admin token, the only possibility which I can think is request directly to Elasticsearch in order to create that account